src/vue/components/morph-parts/morph-data.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div class="alpheios-morph__features">

```

Comment:
classname again -- this seems as if it should be alpheios-morphdata as a top level class, and then alpheios-morph__morphdata below should be something like alpheios-morphdata__lemma

Reply:
Fixed